front/app/modules/commercial/analytics/admin/components/VisitorsCard/Chart/renderTooltip.tsx

```diff
@@ -16,7 +16,7 @@ import { FormattedMessage, MessageDescriptor } from 'utils/cl-intl';
 import { toFullMonth } from 'utils/dateUtils';
 
 // typings
-import { TimeSeriesRow } from '../../../hooks/useVisitorsData/typings';
+import { TimeSeriesRow } from '../../../hooks/useVisitors/typings';
```

Comment:
Also changed the names of hooks so that they don't all end with 'Data'- felt a bit redundant